bin/compile

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+[ "$BUILDPACK_DEBUG" = "true" ] && set -x
```

Comment:
Isn't it better to compare things like that in Bash? I don't remember why though 😅 

```suggestion
[ "x$BUILDPACK_DEBUG" = "xtrue" ] && set -x
```

Reply:
https://unix.stackexchange.com/questions/490393/whats-the-purpose-of-adding-a-prefix-on-both-sides-of-a-shell-variable-comparis

It explains everything :D

Reply:
Thanks, that's a nice write up.
TL;DR: surrounding variables with `"` is enough